src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -120,6 +138,40 @@ class MetaInfoPanelImpl extends React.PureComponent<Props> {
     }
   }
 
+  _handleMoreInfoButtonClick = () => {
+    this.setState((state) => ({ showsMoreInfo: !state.showsMoreInfo }));
+  };
+
+  _renderMoreInfoSection(section: ExtraProfileInfoSection) {
+    return (
+      <div key={section.label}>
+        <h2 className="metaInfoSubTitle" key={'title ' + section.label}>
+          {section.label}
+        </h2>
+        <div className="metaInfoSection" key={'section ' + section.label}>
+          {section.entries.map(({ label, format, value }) => {
+            return (
+              <div className="moreInfoRow" key={label}>
+                <span className="metaInfoWideLabel">{label}</span>
+                <div className="moreInfoValue">
+                  {formatFromMarkerSchema('moreInfo', format, value)}
```

Comment:
The same comment applies here: this isn't a marker so this doesn't look appropriate. However I realize that `formatFromMarkerSchema` is more generic than just for markers and could probably move to another file (`src/utils/format-value.js` for example) and be renamed (`formatValue`).
I would prever that we do it now, in a separate PR, and then rebase this patch on top of it.
However if that's too much work for you now, I think we can go in this way and file a follow-up issue.

Reply:
As explained before, I'm totally for the changes, but I think they are best in a "Factor out marker formatting" PR which I'll submit after the admission of #4201, if that's ok.